recipe/deploy/cleanup.php

```diff
@@ -4,26 +4,17 @@
 // Use sudo in deploy:cleanup task for rm command.
 set('cleanup_use_sudo', false);
 
-desc('Cleanups old releases');
+desc('Cleanup old releases');
```

Comment:
Let’s keep an s at the end. To be in descriptive form.

Reply:
But that's not correct English? You probably mean "Cleans up old releases"

Reply:
Yes! I’m not native English bearer and will need proofread of docs after I finish. 😌

Reply:
Is this okay the way it is, or did you still want me to revert it?